test/exfmt/integration/def_test.exs

```diff
@@ -11,14 +11,12 @@ defmodule Exfmt.Integration.DefTest do
   end
 
   test "def with long guard" do
-    """
-    def one?(x) when x in [:one, "one", 1, "1"] do
-      true
-    end
-    """ ~> """
-    def one?(x)
-        when x in [:one, "one", 1, "1"] do
-      true
+    assert_format """
+    defp valid?(left, doc, right)
+         when is_doc(left)
+         when is_doc(doc)
+         when is_doc(right) do
```

Comment:
Not totally sure I understand. The test needs multiple ```when``` statements to verify alignment. Or do you mean the test example function is confusing?

BTW, I have a pull request coming soon that handles very complex guards with multiple ```when```, ```and```, and ```or``` predicates.